src/Router/buildServerApp.tsx

```diff
@@ -34,21 +36,13 @@ export function buildServerApp(config: AppConfig): Promise<ServerResolveProps> {
       })
 
       if (redirect) {
-        resolve({
-          redirect,
-          status,
-        })
+        resolve({ redirect, status })
 
         return
       }
 
       const bootProps = {
-        system: {
-          ...config,
-          relayEnvironment,
-          resolver,
-          routes,
-        },
+        system: { ...config, relayEnvironment, resolver, routes, currentUser },
```

Comment:
Now we have SSR rendering of the follow buttons in a 'followed' state!

Reply:
Very nicee